tests/Agent/IntegrationTests/UnboundedIntegrationTests/CosmosDB/CosmosDBTests.cs

```diff
@@ -198,7 +198,6 @@ public void BulkCreatingItemsTests()
                 new Assertions.ExpectedMetric { metricName = $"Datastore/operation/CosmosDB/ReadDatabase", metricScope = expectedTransactionName, callCount = 1 },
                 new Assertions.ExpectedMetric { metricName = $"Datastore/operation/CosmosDB/DeleteDatabase", metricScope = expectedTransactionName, callCount = 1 },
                 new Assertions.ExpectedMetric { metricName = $"Datastore/operation/CosmosDB/CreateCollection", metricScope = expectedTransactionName, callCount = 1 },
-                new Assertions.ExpectedMetric { metricName = $"Datastore/statement/CosmosDB/{_testContainerName}/BatchDocument", metricScope = expectedTransactionName, callCount = 1 }
```

Comment:
Sometimes this call isn't triggered, but I'm not sure why. We can add a tech debt ticket to investigate if anyone is concerned.

Reply:
Do we think this is possibly related to an instrumentation bug?

Reply:
I don't think so. The test seems to be performing operations in two concurrent tasks and hoping they get batched. There is actual batching functionality in the library that we aren't using for whatever reason. I'll put in a task to revisit. Maybe we can make the test more reliable.

Reply:
https://github.com/newrelic/newrelic-dotnet-agent/issues/1831